sklearn/linear_model/least_angle.py

```diff
@@ -809,6 +809,11 @@ class Lars(MultiOutputMixin, RegressorMixin, LinearModel):
         setting ``fit_path`` to ``False`` will lead to a speedup, especially
         with a small alpha.
 
+    jitter : float, default=0.0001
+        Uniform noise parameter, added to the y values, to satisfy \
+        the model's assumption of one-at-a-time computations \
+        (Efron et al. 2004).
```

Comment:
\ are not necessary at the end of the lines.
Also I would not have this by default as 1e-4 can be big for certain y depending on scale.